content/site.json

```
@@ -68,6 +68,7 @@
     }
   ],
   "announcement": "",
+  "devPreviewAnnouncement": "You are visiting the Pontus-X Developer Preview. Please note that some functionalities are in beta and may not work as expected.",
```

suggestion: I think this supports markdown, so we could add a link to main version:

```suggestion
  "devPreviewAnnouncement": "You are visiting the Pontus-X Developer Preview. Please note that some functionalities are in beta and may not work as expected. Visit the [Pontus-X portal](https://pontus-x.eu) instead.",
```

If it does not support md, we can skip this